embedded-database-spring-test/src/test/java/io/zonky/test/db/config/ConfigurationPropertiesIntegrationTest.java

```diff
@@ -51,7 +51,7 @@ public void testConfigurationProperties() throws Exception {
 
         JdbcTemplate jdbcTemplate = new JdbcTemplate(dataSource);
 
-        String collate = jdbcTemplate.queryForObject("show lc_collate", String.class);
+        String collate = jdbcTemplate.queryForObject("SELECT datcollate FROM pg_database WHERE datname NOT IN('template0', 'template1', 'postgres') LIMIT 1", String.class);
```

Comment:
I'd like to keep the test code as simple as possible. So, unless this change is necessary for some reason, I would prefer the original version.